### PR TITLE
Improve Shaders

### PR DESCRIPTION
**Pull request by tinxx**
-- Due to an oversight on my part, this MR is based on #804 – so please wait until #804 is merged. --

This MR improves the portal shaders on various aspects:

- The inside of clipped nodes is visible.
- Text shaders honor portals again
  -  The updated coordinates are used for the portal effect after billboarding is applied so that the portal effect now works as expected.
- Both points that define the portal rect are stored in one shader property.
  - Shader `Vector` properties always use four dimensions, so it is more efficient to store both 2D coords in one property.
- Files using both tabs and spaces for indentations have been changed to use either one (but not both).

### CODE REVIEW

File: Assets/Resources/Shaders/TMP_SDF-Mobile Portal Overlay.shader
Comment (falko17) [inline Assets/Resources/Shaders/TMP_SDF-Mobile Portal Overlay.shader:156]: There's still some mixed tabs/spaces here.

File: Assets/Resources/Shaders/TMP_SDF-Mobile Portal.shader
Comment (falko17) [inline Assets/Resources/Shaders/TMP_SDF-Mobile Portal.shader:154]: See above: Mixed spaces/tabs.

File: Assets/Resources/Shaders/TransparentLinePortalShader.shader
Comment (falko17) [inline Assets/Resources/Shaders/TransparentLinePortalShader.shader:99]: Do we still need this line, now that we no longer use the alpha value to hide fragments outside the portal?

File: Assets/SEE/Game/Portal.cs
Comment (falko17) [inline Assets/SEE/Game/Portal.cs:83]: This isn't part of your changes, but the parameter documentation does not match the actual parameter.

**Review by falko17 — CHANGES_REQUESTED** [review]
Just four small comments this time. Honestly, it makes a lot of sense to use a single property for the portal (since it has four components anyway), I don't know why we didn't do it like this before.

File: Assets/Resources/Shaders/TransparentLinePortalShader.shader
Comment (tinxx) [reply Assets/Resources/Shaders/TransparentLinePortalShader.shader:99]: I thought it was part of the alpha blending. Transparency can be applied via texture.

If you know more, or want to spend some time investigating, please feel free :)

Comment (falko17) [reply Assets/Resources/Shaders/TransparentLinePortalShader.shader:99]: > I thought it was part of the alpha blending. Transparency can be applied via texture.

That makes sense, I forgot that the texture can apply transparency too, so let's keep it as-is.

**Comment by falko17** [comment]
This PR should also be ready now, but I'll wait until #804 is merged before approval, so that we don't accidentally merge before then.

**Review by falko17 — APPROVED** [review]

**Pull request merged**
